### PR TITLE
Cv fixes

### PR DESCRIPTION
**Pull request by fategg555**
merging comp code into main

DISCARD STASHED CHANGES

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Vision.java [hunk 1]
```diff
@@ -57,6 +58,8 @@ public void periodic() {
         Units.degreesToRadians(result.getBestTarget().getPitch()))
         );
       targetAngle = result.getBestTarget().getYaw();
+      SmartDashboard.putNumber("distance", targetDistance);
```
Comment (ryan-huang1) [inline src/main/java/frc/robot/subsystems/Vision.java:61]: do we want to keep these in main code? I don't think operators are going to much debugging during a match. we should try to keep shuffleboard kinda organized imo

File: src/main/java/frc/robot/LED.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        led.start();
+        led.stop();
+    }
+
+    public void startLED() {
+>>>>>>> Stashed changes
+        led.start();
+    }
+
+    // public void setRed() {
```
Comment (ryan-huang1) [inline src/main/java/frc/robot/LED.java:41]: can this comment block be deleted?

File: src/main/java/frc/robot/LED.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import frc.robot.Constants.LEDPreset;
+
+public class LED {
```
Comment (ryan-huang1) [inline src/main/java/frc/robot/LED.java:11]: didn't drive team wants some extra lighting modes, should we add those before we merge into main?

**Review by ryan-huang1 — DISMISSED** [review]

File: src/main/java/frc/robot/LED.java (hunk 2 above)
Comment (fategg555) [reply src/main/java/frc/robot/LED.java:41]: > DISCARD ALL STASHED CHANGES

File: src/main/java/frc/robot/LED.java (hunk 3 above)
Comment (fategg555) [reply src/main/java/frc/robot/LED.java:11]: This is working comp code, we can develop new features in different branch and merge after testing.

File: src/main/java/frc/robot/subsystems/Vision.java (hunk 1 above)
Comment (fategg555) [reply src/main/java/frc/robot/subsystems/Vision.java:61]: We can clean up before guilford

**Review by JakeBoggs — APPROVED** [review]

**Review by JakeBoggs — APPROVED** [review]

**Pull request merged**